account_vat_period_end_statement/views/account_view.xml

```diff
@@ -0,0 +1,185 @@
+                                        <field name="account_id" />
+                                        <field name="amount" />
+                                   </tree>
+                                    <form>
+                                        <group>
+                                            <field name="tax_id" />
+                                            <field name="account_id" />
+                                            <field name="amount" />
+                                        </group>
+                                   </form>
+                                </field>
+                            </group>
+                            <group col="3" colspan="4">
+                                <separator colspan="3" string="Previous Credits VAT"/>
+                                <field name="previous_credit_vat_account_id" attrs="{'required':[('previous_credit_vat_amount','!=',0)]}"/>
+                                <field name="previous_credit_vat_amount" nolabel="1"/>
+                                <separator colspan="3" string="Previous Debits VAT"/>
+                                <field name="previous_debit_vat_account_id" attrs="{'required':[('previous_debit_vat_amount','!=',0)]}"/>
+                                <field name="previous_debit_vat_amount" nolabel="1"/>
+                                <separator colspan="3" string="Due interests"/>
+                                <field name="interests_debit_vat_account_id" attrs="{'required':[('interests_debit_vat_amount','!=',0)]}"/>
+                                <field name="interests_debit_vat_amount" nolabel="1"/>
+                                <separator colspan="3" string="Tax credits"/>
+                                <field name="tax_credit_account_id" attrs="{'required':[('tax_credit_amount','!=',0)]}"/>
+                                <field name="tax_credit_amount" nolabel="1"/>
+                            </group>
+                            <separator colspan="3" string="Other VAT Credits / Debits or Tax Compensations (Positive amounts for credits, negative for debits)"/>
+                            <field name="generic_vat_account_line_ids" colspan="3" nolabel="1">
+                                <tree editable="bottom" string="Account Lines">
+                                    <field name="account_id"/>
+                                    <field name="name" />
+                                    <field name="amount" />
+                               </tree>
+                                <form >
+                                    <field name="account_id" />
+                                    <field name="name" />
+                                    <field name="amount" />
+                               </form>
+                            </field>
+                        </group>
+                        <group>
+                            <field name="state"/>
```

Comment:
`state` è già presente alla riga 19

Reply:
Rimosso in https://github.com/OCA/l10n-italy/compare/38379f047bfaa2dbb2805a7af4ed9962e8bef6bf..9c7f4963ee6a4271dacb7c97ee4049dd2973a484